conftest.py

```diff
@@ -435,6 +435,25 @@ def small_client(
             yield client
 
 
+def _cluster_memory(client: distributed.Client) -> int:
+    "Total memory available on the cluster, in bytes"
+    return int(
+        sum(w["memory_limit"] for w in client.scheduler_info()["workers"].values())
+    )
```

Comment:
Seems unfortunate to delete the utility function such that it's inaccessible to tests who might want to do something different with the cluster memory measurement.

Reply:
right, I'll just rename it to `get_cluster_memory`